c_src/Makefile

```
@@ -235,10 +234,14 @@ $(C_SRC_ENV):
 		io_lib:format( \
 			\"ERTS_INCLUDE_DIR ?= ~s/erts-~s/include/~n\" \
 			\"ERL_INTERFACE_INCLUDE_DIR ?= ~s~n\" \
-			\"ERL_INTERFACE_LIB_DIR ?= ~s~n\", \
+			\"ERL_INTERFACE_LIB_DIR ?= ~s~n\" \
+			\"ERL_INTERFACE_LIBS ?= ~s -lei~n\", \
 			[code:root_dir(), erlang:system_info(version), \
 			code:lib_dir(erl_interface, include), \
-			code:lib_dir(erl_interface, lib)])), \
+			code:lib_dir(erl_interface, lib), \
+			try {match, _} = re:run(erlang:system_info(otp_release), \
+			\"^2[3-9]$$\"), \"\" catch _:_ -> \"-lerl_interface\" end \
```

Тут вроде бы и более привычный case expresision должен справиться?
```suggestion
			case re:run(erlang:system_info(otp_release), \"^2[3-9]$$\") of \
			    {match, _} -> \"\"; \
			    nomatch -> \"-lerl_interface\" \
			end \
```